projectgenerator/libili2db/ili2dbconfig.py

```diff
@@ -161,6 +162,9 @@ def to_ili2db_args(self, hide_password=False):
         if self.ilimodels:
             args += ['--models', self.ilimodels]
 
+        if self.tomlfile:
+            args += ["--iliMetaAttrs", self.tomlfile]
```

Comment:
@marioba do you remember, is this parameter for all ili2db commands, or only schemaimport?